src/views/List.jsx

```diff
@@ -8,9 +8,10 @@ export function List({ data }) {
 			</p>
 			<ul>
 				{data && data.length > 0 ? (
-					data.map((item) =>
-						item.items.map((name, id) => <ListItem key={id} name={name} />),
-					)
+					data.map((item, id) => (
+						// Removed double map to reflect current Firestore data model
```

Comment:
This would be great as a GitHub comment, not necessarily a code comment!

Reply:
This is helpful, will remove thanks!